samples/CommunityToolkit.Maui.Sample/Platforms/Android/AndroidManifest.xml

```diff
@@ -2,4 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true"></application>
 	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
This isn't a good practice. Because if the app doesn't use this feature and reference our lib this permission will be added to their app, and can be refused by app stores.